xmake.lua

```diff
@@ -14,4 +14,26 @@ target("EngineSquared")
 
     if is_mode("debug") then
         add_defines("DEBUG")
-    end
\ No newline at end of file
+    end
+
+for _, file in ipairs(os.files("tests/*.cpp")) do
```

Comment:
Does it work for cpp files which are in folders inside the test folder?

Reply:
didnt test

Reply:
fixed